internal/venice-client-common/src/main/java/com/linkedin/venice/stats/VeniceMetricsConfig.java

```diff
@@ -166,6 +196,18 @@ public class VeniceMetricsConfig {
   private final int otelExponentialHistogramMaxScale;
   private final int otelExponentialHistogramMaxBuckets;
 
+  /**
+   * Config to enable pre-compute Attributes for Venice metrics.
+   * check {@link #OTEL_VENICE_PRE_CREATE_ATTRIBUTES} for more details
+   */
+  private final boolean preCreateAttributes;
+
+  /**
+   * Config to enable lazy initialization of Attributes and caching them.
+   * check {@link #OTEL_VENICE_LAZY_INITIALIZE_ATTRIBUTES} for more details
+   */
+  private final boolean lazyInitializeAttributes;
```

Comment:
These two should be mutually exclusive strategies, right? Shall we use a single config option, so that users can specify different strategies?
Honestly-speaking, having 3 modes is an overkill IMO, and lazy-init should be good enough and in theory the regular test should be able to certify it.